### PR TITLE
Use os.path.exists() to check if files exist

### PR DESCRIPTION
**Pull request by JonEsparaz**
I noticed an incorrect filename for an image in https://github.com/themeetinghouse/web/commit/2887f8714b3a325f68bf4fa1f4a032b8a9244ff9, which can easily be avoided by checking if the file exists.

I've fixed all the incorrect filenames currently found in the JSON files.

Edit: I've added tests for the PDFs too.

### CODE REVIEW

File: public/static/content/about-us.json [hunk 1]
```diff
@@ -129,7 +129,7 @@
           {
             "type": "link",
             "title": "Finance Update - October 2020",
-            "navigateTo": "/static/pdfs/annual-reports/Finance Update - October 2020_v2.pdf",
+            "navigateTo": "/static/pdfs/annual-reports/Finance Update - October 2020.pdf",
```
Comment (JonEsparaz) [inline public/static/content/about-us.json:132]: Check if PDF is correct.

File: public/static/content/annual-reports.json [hunk 2]
```diff
@@ -31,89 +30,89 @@
           {
             "title": "2019",
             "text": "Summary of 2019 Report",
-            "imageSrc": "/static/images/homepage-4-1.jpg",
+            "imageSrc": "/static/images/homepage-4-1-new.jpg",
             "imageAlt": "Young woman sitting at a table and looking at her notes from Sunday teaching at The Meeting House.",
-            "url": "/static/pdfs/annual-reports/xxxxx"
+            "url": "/static/pdfs/annual-reports/TMH_2019AnnualReport_Final_v8_NoCuts_NoBleeds.pdf"
```
Comment (JonEsparaz) [inline public/static/content/annual-reports.json:35]: Check if PDF is correct.

File: public/static/content/annualreport2020.json [hunk 3]
```diff
@@ -177,7 +177,7 @@
           {
             "type": "link",
             "title": "Download PDF",
-            "navigateTo": "/static/pdfs/annual-reports/TMH_2020AnnualReport_v6.pdf"
+            "navigateTo": "/static/pdfs/annual-reports/2020_Annual_Report_combined.pdf"
```
Comment (JonEsparaz) [inline public/static/content/annualreport2020.json:180]: Check if PDF is correct.

File: public/static/content/gostorytelling.json [hunk 4]
```diff
@@ -164,13 +164,6 @@
           }
         ]
       },
-      {
```
Comment (JonEsparaz) [inline public/static/content/gostorytelling.json:167]: Donor-FAQs-for-Go.pdf does not exist. I've removed the link.

File: public/static/data/redirect.json [hunk 5]
```diff
@@ -343,10 +343,6 @@
     "id": "mccupdate",
     "to": "https://mcc.zoom.us/meeting/register/tJAucemvpjgtH9fBdmhRa8LwUgXPdBJr1dsu"
   },
-  {
-    "id": "communion",
-    "to": "/static/pdfs/kidmax-files/Family-Guide-Communion.pdf"
```
Comment (JonEsparaz) [inline public/static/data/redirect.json:348]: Family-Guide-Communion.pdf does not exist. I've removed the redirect.

**Comment by JonEsparaz** [comment]
@lucastbelem please see the above five comments. I need your help confirming a few things with the PDFs.

File: public/static/content/gostorytelling.json [hunk 6]
```diff
@@ -164,13 +164,6 @@
...
-        "type": "goLink",
-        "style": "link",
-        "title": "Go Campaign FAQ",
-        "navigateTo": "/static/pdfs/go/Donor-FAQs-for-Go.pdf",
```
Comment (JonEsparaz) [inline public/static/content/gostorytelling.json:171]: Donor-FAQs-for-Go.pdf does not exist. I've removed the link.

**Comment by lucastbelem** [comment]
@JonEsparaz everything looks good!

**Comment by JonEsparaz** [comment]
Great, thank you @lucastbelem!

**Pull request merged**